data/json/recipes/weapon/bashing.json

```
@@ -157,10 +157,37 @@
     "category": "CC_WEAPON",
     "subcategory": "CSC_WEAPON_BASHING",
     "skill_used": "fabrication",
+    "difficulty": 2,
+    "skills_required": [ "melee", 1 ],
+    "time": "45 m",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "proficiencies": [ { "proficiency": "prof_carpentry_basic", "fail_multiplier": 1 } ],
```

I think you wanted to use carving rather than carpentry, since the cudgel does not require any nails or real carpentry, but a good carving.
```suggestion
    "proficiencies": [ { "proficiency": "prof_carving", "fail_multiplier": 1 } ],
```

---

oh, good catch, i missed that earlier; i was recommending carving in the original discord conversation and didn't review that closely.

---

Yeah, I was reading the comments so when I spotted "carpentry" I was pretty surprised haha.